### ISSUE TITLE
No way to access total amount of labor or labor invested at a work party through laws

### ISSUE THREAD

**Issue by DanielKispert** [StrangeLoopGames/EcoIssues#21677]
While you can get the payment amount and percent of work invested in a work party for laws, you can not get the actual amount of work that was put into a work party, neither through "work party completed" or "worked for work party".

This information would be great to have so we can do more laws on work parties e.g. incur a flat tax per labor invested or count the total amount of labor done for work parties in a custom stat for analysis.

**Comment by DanielKispert** [comment]
more info on this:
these are the values that "worked on work party" gives, which are just payment amount and percentage of work, not absolute labor
![grafik](https://user-images.githubusercontent.com/34270661/116832033-547e6f80-abb3-11eb-9f13-469274c86297.png)
"posted work party" also has just the amount of currency and percent done, nothing about the absolute amount of labor in it
![grafik](https://user-images.githubusercontent.com/34270661/116832057-74159800-abb3-11eb-8532-782ef4304bba.png)
same for "completed work party"
![grafik](https://user-images.githubusercontent.com/34270661/116832070-8263b400-abb3-11eb-82d0-ead0f3b66af1.png)

it would be great to access the labor in the work party somehow

**Comment by ThePiachu** [comment]
Yeah, we wanted to implement a work party tax based on the amount of labour and skill involved, but we can't.

**Comment by IronyTroll** [comment]
This is also an issue we have been experiencing. We would like to pay the person who performed the work party based on the amount of work they have contributed but there is no way for us to read the amount of calories spent.
